plenum/server/query_req_handler.py

```diff
@@ -0,0 +1,17 @@
+from plenum.common.request import Request
+
+from plenum.server.req_handler import RequestHandler
+
+
+class QueryRequestHandler(RequestHandler):
```

Comment:
This is the same as `ReadRequestHandler`.
Please remove this class

Reply:
ok